packages/import-ynab4/importer.js

```diff
@@ -2,8 +2,8 @@ const d = require('date-fns');
 const normalizePathSep = require('slash');
 const uuid = require('uuid');
 const AdmZip = require('adm-zip');
-const actual = require('@actual-app/api/methods');
-const { amountToInteger } = require('@actual-app/api/utils');
+const actual = require('@actual-app/api');
+const amountToInteger = actual.utils.amountToInteger;
```

Comment:
Why this change?

Reply:
I get the below error with the original code.

```
ERROR in ./packages/import-ynab4/importer.js
Module not found: Error: Can't resolve '@actual-app/api/methods' in '/home/tom/Programming/actual/packages/import-ynab4'
 @ ./packages/import-ynab4/importer.js 21:15-49
 @ ./packages/loot-core/src/server/main.js
```

I imagine that it's yarn starting enforcing that you go in through the main entrypoint but I didn't dig too much into it.